ocfweb/main/staff_hours.py

```diff
@@ -9,6 +9,8 @@
 
 @periodic(60, ttl=86400)
 def get_staff_hours():
+    print('IN OCFWEBOCFWEB:')
+    print(real_get_staff_hours())
     return real_get_staff_hours()
```

Comment:
please also put up your ocflib changes so both can be merged at the same time if they depend on one another